src/schema/v2/FeaturedLink/createFeaturedLinkMutation.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  GraphQLString,
+  GraphQLUnionType,
+  GraphQLObjectType,
+  GraphQLBoolean,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { GraphQLNonNull } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { FeaturedLinkType } from "./featuredLink"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+
+interface GravityInput {
+  description: string
+  title: string
+  href: string
+  subtitle?: string
+  display_on_mobile?: boolean
+  display_on_desktop?: boolean
```

Comment:
Based on this comment on the [product brief](https://www.notion.so/Forque-Site-tab-migration-505dc19606c84c1e8d1230e0f5065d61?d=065c8375c6c940d5bbbd15ed22760074#e5b5817597e146e783a17b2e4e947170), it looks like we technically don't need to support these.

Reply:
Specifically the `display_on...` fields

Reply:
Cool, will remove. Those fields default to `true` (and ofc unclear if they are even used anywhere anyway).